NAMESPACE

```diff
@@ -3,6 +3,7 @@
 export(bin)
 export(closest)
 export(common)
+export(join)
```

Comment:
I know you like short function names, but here I would rather call it `joinNumeric` instead - this function will in fact only work on `numeric` vectors and we don't run into the `join` function from tidyverse that joins `tibbles` and other things.

Reply:
In fact I don't have a problem to run `sed 's/join/joinNumeric/g'` but isn't it the same discussion we had with `bin`?

Reply:
Yep, same discussion 😄  - my reason to call it `joinNumeric` was that from a `join` function I would rather expect to join something two-dimensional (i.e. `matrix` or `data.frame`) - similar to the SQL call. That's why I called it *Numeric* to specifically state that this function instead is going to require `numeric` as input.

That was my reasoning - I leave it now to you how you want to call it.

Reply:
Ok, I will keep `join` for now. I promise I will rename it if this is causing problems/confusion.